### DISCUSSION TITLE
请问两种部署方式会有性能差异吗？

### DISCUSSION THREAD

**Discussion by AlPacino233** [criyle/go-judge#126] [Q&A] [answered]
我这边两台服务器，部署方式不同，一台是下载对应文件编译运行的，一台是docker部署的，现在我两台服务器判题时提交相同的代码， 输入测试点也是一样，docker部署运行时间要12s左右，编译运行的只需要5s，这种情况是部署差异导致的吗？还是说docker部署的时候有什么配置，限制了使用率

**Comment by AlPacino233** [comment]
docker部署的那台服务器硬件规格甚至比另一台要高

**Reply by criyle** [reply, marked as answer]
也要考虑 docker 是怎么部署的，比如 docker desktop 是虚拟机而不是容器
